tests/integration/test_models/standard_model_tests.py

```diff
@@ -71,6 +71,10 @@ def test_solving(self, solver=None, t_eval=None):
         # Overwrite solver if given
         if solver is not None:
             self.solver = solver
+        else:
```

Comment:
should this be else or just default behaviour?

Reply:
yeah probably should just be default. 

I agree, I think a separate repository on pybamm-team with paper results and any undocumented (or poorly documented) examples would be good. Then make sure the examples that come as part of pybamm are actually really useful for learning how to use it/showcasing feature, rather than just a relatively random collection of scripts we make during development.